RUDP/src/net/rudp/ReliableSocket.java

```diff
@@ -48,7 +48,6 @@
 import java.util.ArrayList;
 import java.util.Iterator;
 import java.util.Random;
-
```

Comment:
I generally like having blank lines between imports that belong to separate libraries  - many IDE's also have a feature to auto-ident in such a way.